includes/class-wp-statistics-user.php

```diff
@@ -68,6 +68,20 @@ public static function get($user_id = false)
         return $user_info;
     }
 
+
+
+    public static function getMeta($metaKey, $single = false, $userId = false)
+    {
+        $userId = !empty($userId) ? $userId : get_current_user_id();
+        return get_user_meta($userId, $metaKey, $single);
+    }
+
+    public static function saveMeta($metaKey, $metaValue, $userId = false)
+    {
+        $userId = !empty($userId) ? $userId : get_current_user_id();
+        return update_user_meta($userId, $metaKey, $metaValue);
```

Comment:
Maybe sanitize here?

Reply:
Actually, the `update_user_meta `function sanitizes values internally. @mostafasoufi